hack/.golint_failures

```diff
@@ -559,6 +559,7 @@ staging/src/k8s.io/sample-apiserver/pkg/apis/wardle/v1alpha1
 staging/src/k8s.io/sample-apiserver/pkg/registry/wardle/fischer
 staging/src/k8s.io/sample-apiserver/pkg/registry/wardle/flunder
 test/e2e/common
+test/e2e/framework/kubelet
```

Comment:
Why are we introducing lint failures in a refactor?

Reply:
@cblecker Updated, PTAL